qualtran/bloqs/rotations/hamming_weight_phasing_test.py

```diff
@@ -54,7 +54,7 @@ def test_hamming_weight_phasing(n: int, theta: float):
 class TestHammingWeightPhasingViaPhaseGradient(GateWithRegisters):
     bitsize: int
     exponent: float
-    eps: int = 1e-2
```

Comment:
What was going on here, was this just a typo?

Reply:
using `int` instead of `float` was a typo and the default value wasn't really used since all callsites were passing `eps` explicitly. So yes; It was just a typo.